### PR TITLE
Feature/articles bulletins census

### PR DESCRIPTION
**Pull request by yomiodeleye**
### What

Adding a tag to Articles/Bulletins to GTM for them to filter if a page have census flag

### Who can review

FD

### CODE REVIEW

File: mapper/mapper.go [hunk 1]
```diff
@@ -362,7 +382,7 @@ func CreateBulletinModel(basePage coreModel.Page, bulletin articles.Bulletin, bc
 
 	currentUrl := getCurrentUrl(requestProtocol, model.SiteDomain, model.URI, lang)
 	model.ShareLinks = createShareLinks(model.Metadata.Title, currentUrl)
-
+	model.PreGTMJavaScript = createPreGTMJavaScript(model.Metadata.Title, model)
```
Comment (rafahop) [inline mapper/mapper.go:385]: Can we add an assertion in the unit test to check this value?

**Review by rafahop — COMMENTED** [review]
There is an audit failure that needs fixing

File: mapper/mapper_test.go [hunk 2]
```diff
@@ -510,6 +510,7 @@ func TestUnitMapper(t *testing.T) {
 				So(model.Census2021, ShouldEqual, true)
 				So(model.AboutTheData, ShouldEqual, true)
 				So(len(model.Sections), ShouldEqual, len(bulletin.Sections))
+				So(model.PreGTMJavaScript, ShouldNotBeEmpty)
```
Comment (rafahop) [inline mapper/mapper_test.go]: Can we check the value please?

Comment (yomiodeleye) [reply mapper/mapper_test.go]: i am not testing for this now.

File: mapper/mapper_test.go [hunk 3]
```diff
@@ -481,6 +481,7 @@ func TestUnitMapper(t *testing.T) {
 			Convey("CreateBulletinModel maps correctly", func() {
 				requestProtocol := "https"
 				model := CreateBulletinModel(basePage, bulletin, breadcrumbs, "cy", requestProtocol, serviceMessage, bannerData)
+				preGTMJavaScript := createPreGTMJavaScript(model.Metadata.Title, model)
```
Comment (rafahop) [inline mapper/mapper_test.go]: Isn't this calling actual code to create the expected response?? There is not value in that, it isn't testing anything?

**Review by rafahop — APPROVED** [review]

**Pull request merged**